### PR TITLE
Extend readme to capture `NETWORK` variable

### PR DESCRIPTION
**Pull request by da-kami**
Since I ran into this problem I thought I'll capture this in the readme :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,21 @@ note: Flutter might ask you which target you'd like to run.
...
+To run on `regtest` you have to change the default network (`testnet`) to `regtest` by setting the `NETWORK` environment variable.
+
+Example command for maker:
+
+`NETWORK=regtest make maker`
+
+Example command for taker:
+
+`NETWORK=regtest flutter run`
+
```
Comment (klochowicz) [inline README.md:94]:
```suggestion

You might want to consider adding the following line to your .bashrc/.zshrc:

`export NETWORK=regtest`

in order to always run on regtest.
```

**Review by klochowicz — APPROVED** [review]
looks good. I added a suggestion that can make it easier for devs.

File: README.md [hunk 2]
```diff
@@ -80,7 +80,21 @@ note: Flutter might ask you which target you'd like to run.
...
+Note that the `iOS` simulator will _not_ pick up the envonriment variable, so make sure you run on a `native` target when testing with regtest!
```
Comment (luckysori) [inline README.md:95]:
```suggestion
Note that the `iOS` simulator will _not_ pick up the environment variable, so make sure you run on a `native` target when testing with regtest!
```

**Review by luckysori — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (da-kami) [reply README.md:94]: I think that heavily depends on your setup - I don't really like to make these things global :)

**Pull request merged**